content/Watsonx/WatsonxAI/100.md

```diff
@@ -15,6 +15,89 @@ toc: true
 
 ## FAQs
 
+## How do I access a watsonx.ai instance?
+
+### How to acquire a watsonx.ai instance from Techzone (TZ)
+
+
+To gain access to a watsonx.ai instance from TZ, navigate to this link [here](https://ibm.biz/WATSONX-AI-TZ). Once there:
+
+1. Select **Reserve now**
+     ![](images/100/reserve-now-btn.png)
+
+2. Change the Name of the reservation to *your name* watsonx.ai L3 SaaS (1).
```

Comment:
This instruction includes 'L3' in the name, but the image does not have 'L3' in the name.